LayoutTests/http/tests/site-isolation/pointer-lock.html

```diff
@@ -0,0 +1,39 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+
+<head>
+    <script src="/js-test-resources/js-test.js"></script>
+    <script src="/js-test-resources/ui-helper.js"></script>
+    <meta name="fuzzy" content="maxDifference=112; totalPixels=900" />
+</head>
+<script>
+    if (window.testRunner)
+        testRunner.waitUntilDone();
+
+    window.addEventListener('message', (event) => {
+        if (event.data.type === 'pointerLockChanged') {
+            console.log('Iframe pointer lock changed:', event.data.locked);
+
+            // Verify site isolation - parent should NOT have pointer lock
+            const parentLocked = document.pointerLockElement !== null;
+            console.log('Parent document locked:', parentLocked);
+
+            if (event.data.locked && !parentLocked) {
+                console.log('Site isolation working correctly');
+            }
+
+            if (window.testRunner)
+                testRunner.notifyDone();
+        }
+    });
+
+    // Pointer lock not supported on iOS
+    const hasPointerLock = 'pointerLockElement' in document;
+    if (!hasPointerLock) {
+        console.log("Pointerlock not supported");
+        testRunner.notifyDone();
```

Comment:
Should probably check `if (window.testRunner)` here for consistency with the other places.

Reply:
Ah, right missed that one.